c/alloc.c

```diff
@@ -525,7 +543,9 @@ static char *safeMalloc64Internal(int size, char *site, long long token){
   if (res != NULL){
     memset(res,0,size);
   } else{
-    printf("MALLOC failed, got NULL for size %d at site %s\n",size, site);
+    if (MALLOC_TRACE_LEVEL >= 1){
+      printf("MALLOC failed, got NULL for size %d at site %s\n",size, site);
```

Comment:
This just may not be needed at all, because you will know success by getting a return of NULL.